frontend/packages/kubevirt-plugin/src/selectors/vmi/basic.ts

```diff
@@ -15,3 +15,11 @@ export const getVMIConditionsByType = (
   const conditions = vmi && vmi.status && vmi.status.conditions;
   return (conditions || []).filter((cond) => cond.type === condType);
 };
+
+export const getVmiTemplateLabels = (vmi: VMIKind): { [key: string]: string } =>
+  (vmi &&
```

Comment:
could we use this format instead?

```tsx
_.get(vmi, 'spec.template.metadata') && vmi.spec.template.metadata.labels
```

it is more readable and takes less lines

Reply:
good idea, simple as with `lodash` but still type-checked